src/mscorlib/src/System/Span.cs

```diff
@@ -285,6 +283,25 @@ public void Set(ReadOnlySpan<T> values)
 
             SpanHelper.CopyTo<T>(ref DangerousGetPinnableReference(), ref values.DangerousGetPinnableReference(), values.Length);
         }
+
+        /// <summary>
+        /// Fills the contents of this span with the value.
+        /// </summary>
+        /// <param name="value">The value to fill the span content with.</param>
+        public void Fill(T value)
+        {
+            if (_length > 0)
+                SpanHelper.Fill<T>(ref DangerousGetPinnableReference(), value, _length);
+        }
+
+        /// <summary>
+        /// Fills the contents of this span with the default value.
+        /// </summary>
+        public void Clear()
+        {
+            if (_length > 0)
+                SpanHelper.Clear<T>(ref DangerousGetPinnableReference(), _length);
```

Comment:
Removing unsafe modifier is not a good reason. unsafe modifier does not matter - it does not do anything to the public shape of the API.